NewDawn/NewDawn/Models/UserModels.swift

```diff
@@ -258,7 +258,16 @@ class UserProfileBuilder{
     // Usage for user to other users' profiles information from backend
     static func fetchUserProfiles(params: [String:String] = [:], callback: @escaping (NSDictionary, String?) -> Void){
         // TODO: get username and api_key from keychain/local storage
-        let request = UserProfileBuilder.createGetProfileRequest(params)
+        var temp_params = params
```

Comment:
Can you wrap this logic of parsing preference into a static function in user model?